prow/config/postsubmits.yaml

```diff
@@ -418,6 +418,52 @@ postsubmits:
             memory: 1Gi
 
   maistra/istio-operator:
+  - name: update-charts-in-2.0
+    decorate: true
+    path_alias: maistra.io/istio-operator
+    skip_report: false
+    branches:
+    - ^maistra-1.0$
+    - ^maistra-1.1$
+    # Allow for testing
+    - playground
+    labels:
+      preset-github: "true"
+    extra_refs:
+    - base_ref: main
+      org: maistra
+      path_alias: maistra.io/test-infra
+      repo: test-infra
+    spec:
+      containers:
+      - image: "quay.io/maistra-dev/maistra-builder:2.0"
+        imagePullPolicy: Always
+        command:
+        - ../test-infra/tools/automator.sh
+        - -o maistra
+        - -r istio-operator
+        - -b maistra-2.0
+        - -f /creds-github/github-token
+        - -c make clean update-1.0-charts update-1.1-charts
```

Comment:
can you run `make gen` here and in 2.1 job?

Reply:
Done.

Reply:
Do we need `clean`? Probably not, since it's a clean checkout, or isn't it?

Reply:
Done.